src/diffpy/labpdfproc/tests/conftest.py

```diff
@@ -44,4 +47,10 @@ def user_filesystem(tmp_path):
         f.write("good_data.xy \n")
         f.write(f"{str(input_dir.resolve() / 'good_data.txt')}\n")
 
+    user_config_data = {"username": "good_username", "email": "good@email.com"}
+    with open(conf_dir / "test.json", "w") as f:
+        json.dump(user_config_data, f)
+    with open(conf_dir / "test2.json", "w") as f:
```

Comment:
this is not allowed. We will only check for `diffpyconf.json`.

However, we will check for it here, but also in cwd where the programming is running.  We would like to test both situations so we can't write that second file in conftest.py but will have to write it in a bespoke test.